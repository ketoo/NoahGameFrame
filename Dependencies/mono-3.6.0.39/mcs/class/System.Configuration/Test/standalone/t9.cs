using System;
using System.Collections.Generic;
using System.Collections.Specialized;
using System.Text;
using System.Configuration;
using System.Web;

class T1
{
	static void Main(string[] args)
	{
		try
		{
			Configuration config = ConfigurationManager.OpenExeConfiguration (ConfigurationUserLevel.None);
			ConnectionStringsSection sect = config.ConnectionStrings;
			ConnectionStringSettingsCollection connectionstrings = sect.ConnectionStrings;

			connectionstrings.Add (new ConnectionStringSettings ("fromtest", "connectionstringhere"));

			foreach (ConnectionStringSettings cs in connectionstrings) {
				Console.WriteLine ("connectionstring[{0}] = `{1}',`{2}'", cs.Name, cs.ProviderName, cs.ConnectionString);
			}
		}
		catch (Exception e)
		{
			// Error.
			Console.WriteLine(e.ToString());
		}
	}
}
