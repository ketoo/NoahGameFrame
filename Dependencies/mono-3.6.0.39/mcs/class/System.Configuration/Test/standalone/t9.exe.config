<configuration>
<connectionStrings>
	<clear />
	<add name="LocalSqlServer"
		connectionString="data source=127.0.0.1; Integrated Security=SSPI" />
	<add name="AccessFileName"
		connectionString="~/Data/ASPNetDB.mdb" />
</connectionStrings>
</configuration>