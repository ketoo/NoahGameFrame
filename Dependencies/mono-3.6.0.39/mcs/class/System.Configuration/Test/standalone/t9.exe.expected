connectionstring[LocalSqlServer] = `',`data source=127.0.0.1; Integrated Security=SSPI'
connectionstring[AccessFileName] = `',`~/Data/ASPNetDB.mdb'
connectionstring[fromtest] = `',`connectionstringhere'
